salalib/mgraph.cpp

```diff
@@ -1299,6 +1299,30 @@ bool MetaGraph::analyseSegmentsAngular( Communicator *communicator, Options opti
    return retvar;
 }
 
+bool MetaGraph::analyseTopoMetMultipleRadii( Communicator *communicator, Options options ) // <- options copied to keep thread safe
+{
+   m_state &= ~SHAPEGRAPHS;      // Clear axial map data flag (stops accidental redraw during reload)
+
+   bool retvar = false;

```

Comment:
Set retvar to true here - there is no way the `false` set here can be used anywhere, as it will be set to true in the next executed line.